recipes/happly/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "ci.20200822":
```

```suggestion
  "cci.20200822":
```

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases